.github/workflows/deploy_to_snowflake.yml

```diff
@@ -0,0 +1,51 @@
+name: Update Snowflake Normalizer UDF Definition
+
+on:
+    workflow_run:
+        workflows: [deploy_to_pypi]
+        types: [completed]
+jobs:
+  update-snowflake-udf:
+    runs-on: ubuntu-latest
+
+    env:
+      SNOWFLAKE_ACCOUNT: "${{ secrets.SNOWFLAKE_ACCOUNT }}"
+      SNOWFLAKE_USERNAME: "SERVICE_GITHUB_USER"
+      SNOWFLAKE_PRIVATE_KEY: "${{ secrets.SNOWFLAKE_PRIVATE_KEY }}"
+      SNOWSQL_ROLE: "DATA_ENGINEER"
+      SNOWSQL_WAREHOUSE: "GITHUB_WH"
+
+    steps:
+      - uses: actions/checkout@v3
+
+        # Snowflake CLI installation
+      - uses: snowflakedb/snowflake-cli-action@v1.5
+
+          # Use the CLI
+      - name: Execute Snowflake CLI command
+        env:
+            SNOWFLAKE_AUTHENTICATOR: SNOWFLAKE_JWT
+            SNOWFLAKE_USER: ${{ env.SNOWFLAKE_USERNAME }}
+            SNOWFLAKE_ACCOUNT: ${{ env.SNOWFLAKE_ACCOUNT }}
+            SNOWFLAKE_PRIVATE_KEY_RAW: ${{ env.SNOWFLAKE_PRIVATE_KEY }}
+
+        run: |
+          snow sql -x -q 'CREATE OR REPLACE FUNCTION TERAKEET.COMMON.NORMALIZE_URL("URL" VARCHAR)
```

Comment:
nice `snow` command -- does that always fetch the latest from PyPI (assuming yes)